app/serializers/course_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class CourseSerializer < ActiveModel::Serializer
-  attributes :id, :name, :vacancies
+  attributes :id, :name, :vacancies, :inscribed
```

Comment:
The course method must be named with the same name as here. So it'll be inscribed? With the ? mark